airbyte-integrations/connectors/source-hubspot/metadata.yaml

```diff
@@ -42,6 +42,15 @@ data:
         scopedImpact:
           - scopeType: stream
             impactedScopes: ["marketing_emails"]
+      4.0.0:
+        message: >-
+          This update brings extended schema with data type changes for the streams: Deals Property History, Companies Peoperty History.
```

Comment:
This update brings extended schema with data type changes for the streams Deals Property History and Companies Property History. Users will need to refresh their schema and reset their streams after upgrading.

Reply:
updated